modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java

```diff
@@ -1931,85 +1931,76 @@ public void testUpdateObjectEntry() throws Exception {
 			_objectDefinition2.getObjectDefinitionId(), objectEntry.getId(),
 			Arrays.asList("en_US", "fr_FR", "pt_BR"));
 
-		Map<String, String> localizedLongTextObjectFieldI18nValue =
```

Comment:
This SF commit should be squashed with the previous commit; I left comments about a code that does not exist in the final version.